### ISSUE TITLE
Colour temperature 

### ISSUE THREAD

**Issue by Fefe93** [Maassoft/ColorControl#166]
I am unable to change the colour temperature to a warm value on my LG G2. The Colour temperature scale from 0-50 refers only to a cool value and changing the white balance colour temperature to a warm value has no effect.

**Comment by Maassoft** [comment]
So the range is -50 (the warmest temperature) to +50 (the coolest)? If that is the case this will be fixed in the next release.
I could not test this on my C9 because only the white balance color temperature works.

**Comment by Fefe93** [comment]
Yes that's correct, -50 is the warmest and +50 is the coolest. Thanks it would be great if you could fix it as the only way for me to change the colour temperature is through Colorcontrol as the option is greyed out on my LG G2 due to a calibration. Any idea when the next update might be?

**Comment by Fefe93** [comment]
I can see you've fixed this now with the latest update, thank you so much for your good work!
